tests/conftest.py

```diff
@@ -246,9 +242,10 @@ def fixture_rsync_api(cg_context: CGConfig) -> RsyncAPI:
 
 
 @pytest.fixture(name="external_data_api")
-def fixture_external_data_api(cg_context: CGConfig) -> ExternalDataAPI:
+def fixture_external_data_api(analysis_store, cg_context: CGConfig) -> ExternalDataAPI:
     """ExternalDataAPI fixture"""
     _external_data_api: ExternalDataAPI = ExternalDataAPI(config=cg_context)
+    _external_data_api.status_db = analysis_store
```

Comment:
```suggestion
    _external_data_api.status_db = analysis_store
```
or? I might be missing something

Reply:
Not sure if I follow your suggestion. Also not sure why I added this 😅

Reply:
Removed w/o any pytests failing